admin/tim_limitless_consts.php

```diff
@@ -10,16 +10,19 @@
 const TIM_LIMITLESS_POST_HASH_CONTENT_EXCERPT_TITLE = 'tim_limitless_post_hash_content_excerpt_title';
 const TIM_LIMITLESS_STARTUP = "https://trinitymedia.ai/sas/player/amazon_plugin/startup.php";
 const TIM_LIMITLESS_AUDIO_SILENCE = "https://vd.trinitymedia.ai/loading.mp3";
-const TIM_LIMITLESS_POST_HASH_URL = 'https://audio.thetimmedia.com/wordpress/get_posthash';
-const TIM_LIMITLESS_KEYS_URL = 'http://r2v2.memeglobal.com/api/wordpress_campaign?site_domain=';
-const TIM_LIMITLESS_AUDIO_URL = 'https://audio.thetimmedia.com/process';
+
+const TRINITY_AUDIO_SERVICE = 'https://audio.trinityaudio.ai/';
+const TIM_LIMITLESS_POST_HASH_URL = TRINITY_AUDIO_SERVICE . 'wordpress/get_posthash';
```

Comment:
@borisjaskerovich note that the use of `get_` in the APIs here is not required
the point of REST is not using method names in the API. no this can be `wordpress/posthash` (and using POST to create a posthash makes sense) 
and `wordpress/languages` with `GET` is better than `get_languages` etc